pkg/config/data_test.go

```diff
@@ -35,10 +35,11 @@ func TestConfigLoad(t *testing.T) {
 		f *config.File
 	)
 	if err := f.Load("foo", "/path/to/config.toml", c); err != nil {
-		if !errors.As(err, &fsterr.RemediationError{}) {
+		var remediation fsterr.RemediationError
+		if !errors.As(err, &remediation) {
 			t.Errorf("expected RemediationError got: %T", err)
 		}
-		if err.(fsterr.RemediationError).Remediation != fsterr.NetworkRemediation {
+		if remediation.Remediation != fsterr.NetworkRemediation {
 			t.Errorf("expected NetworkRemediation got: %s", err.(fsterr.RemediationError).Remediation)
```

Comment:
Should `err.(fsterr.RemediationError).Remediation` also change to just `remediation`?
```suggestion
			t.Errorf("expected NetworkRemediation got: %s", remediation.Remediation)
```

Reply:
Oops,  yes.